src/components/Ratings/Ratings.st.css

```diff
@@ -0,0 +1,184 @@
+:import {
+    -st-from: "../../common/formatters.st";
+    -st-named: color, fallback, font;
+}
+
+:import {
+    -st-from: "../../common/palette.st.css";
+    -st-named: errorColor;
+}
+
+:vars {
+    /*Defaults*/
+    DefaultTextColor: color-5;
+    DefaultColor: color-8;
+    DefaultEmptyColor: color-3;
+    DisabledColor: color-3;
+    IconGap: 1.5px;
+    IconLargeGap: 5.5px;
+    IconLargeSize: 30px;
+    DefaultFont: font("{theme: 'Body-S', size: '14px', lineHeight: '1.4em'}");
+    LargeFont: font("{theme: 'Heading-M', size: '32px', lineHeight: '1.25em'}");
+}
+
+:vars {
+    /* Overrides */
+    TextColor: --overridable;
+    IconColor: --overridable;
+    IconEmptyColor: --overridable;
+}
+
+.root {
+    -st-states: disabled, error, size( enum(small, large) ), mode( enum(input, display) ), layout( enum(aside, bottom) );
+
+    display: inline-flex;
+    align-items: center;
+    cursor: pointer;
+}
+
+.root:layout(bottom) {
+    flex-direction: column;
+    align-items: baseline;
+}
+
+/* ------ States ------ */
+
+/* SIZE */
+.root:size(large) .icon {
+   padding: 0 value(IconLargeGap);
+}
+
+.root:size(large) .icon svg {
+    width: value(IconLargeSize);
+    height: value(IconLargeSize);
+}
+
+
+.root .iconList {
+    display: flex;
+    align-items: center;
+    flex-direction: row-reverse;
+    flex-wrap: wrap;
+
+    margin: 0 calc(0px - value(IconGap));
+
+    direction: ltr;
+}
+
+.root:size(large) .iconList {
+    margin: 0 calc(0px - value(IconLargeGap));
+}
+
+.root:size(large) .inputOption,
+.root:size(large) .ratingInfo {
+    font: value(LargeFont);
+}
+
+.root:size(large) .icon {
+    margin-top: 7px;
+}
+
+/* DISABLED */
+.root:hover:not(:disabled):not(:mode(display)) .icon path {
```

Comment:
I'm not sure what this rule does, but I am sure that this causes the stars to get blank when hovering over the label.
For instance when I hover over "Achla" all the stars get empty.